IFTPipeline.jl/src/cli.jl

```diff
@@ -493,6 +680,10 @@ function main()
         help = "Preprocess truecolor/falsecolor images"
         action = :command
 
+        "preprocess_tiling_single"
+        help = "Preprocess truecolor/falsecolor images with tile-based histogram equalization"
```

Comment:
The tiling workflow does more than histogram equalization. It also does binarization and ice-water discrimination (ice labeling).